op-node/metrics/metrics.go

```diff
@@ -37,6 +37,9 @@ type Metrics struct {
 	RPCClientRequestDurationSeconds *prometheus.HistogramVec
 	RPCClientResponsesTotal         *prometheus.CounterVec
 
+	L1SourceCache *CacheMetrics
+	// TODO: L2SourceCache *CacheMetrics
```

Comment:
Is this a dead comment?

Reply:
Not dead, it's a hint for the implementer of ENG-2555 (adding caching to L2 source bindings)